pennylane_lightning/src/tests/lightning_util_unittest.cpp

```diff
@@ -57,4 +60,44 @@ INSTANTIATE_TEST_SUITE_P (
                 std::make_tuple(2, 4, 2),
                 std::make_tuple(2, 5, 4)));
 
+
+class getIndicesAfterExclusionTestFixture :public ::testing::TestWithParam<std::tuple<unsigned int, unsigned int>> {
+};
+
+TEST_P(getIndicesAfterExclusionTestFixture, CheckgetIndicesAfterExclusionResults) {
+    unsigned int index = std::get<0>(GetParam());
+    unsigned int qubits = std::get<1>(GetParam());
+
+    vector<vector<unsigned int>> inputs = {
+        {0, 1},
+        {2},
+        {1, 2},
+        {1, 2, 4},
+        {3, 2, 4},
+    };
+
+    vector<vector<unsigned int>> outputs = {
+        {2},
+        {0, 1},
+        {0},
+        {0, 3},
+        {0, 1, 5},
+    };
```

Comment:
Maybe `::testing::Combine` could be used here, see https://stackoverflow.com/a/13240573 and some gate tests in #73